### ISSUE TITLE
optimize for tree shaking

### ISSUE THREAD

**Issue by leviathan-n** [gigobyte/purify#543]
hello 
I'm using this library in a web app with vite as bundler, and notice that I need to import  by `import { xxx } from 'purify-ts/es/xxx'`  to get smallest bundle size.
here is [demo](https://stackblitz.com/edit/vitejs-vite-cvb6u1?file=src/App.ts)

is it possible to remove unused code while using this `import { xxx, yyy } from 'purify-ts` import pattern ?

**Comment by gigobyte** [comment]
Nope, the `es` imports are provided exactly for that reason, the default export is es5 so it's not treeshake-able.
